SEPADirectDebit/src/main/java/com/braintreepayments/api/SEPADirectDebitClient.java

```diff
@@ -56,53 +56,43 @@ public SEPADirectDebitClient(@NonNull Context context, @NonNull String authoriza
      */
     public void createPaymentAuthRequest(@NonNull final SEPADirectDebitRequest sepaDirectDebitRequest,
                                          @NonNull final SEPADirectDebitPaymentAuthRequestCallback callback) {
-        braintreeClient.sendAnalyticsEvent("sepa-direct-debit.selected.started");
-        braintreeClient.sendAnalyticsEvent("sepa-direct-debit.create-mandate.requested");
+        braintreeClient.sendAnalyticsEvent(SEPADirectDebitAnalytics.TOKENIZE_STARTED);
         sepaDirectDebitApi.createMandate(sepaDirectDebitRequest,
                 braintreeClient.getReturnUrlScheme(),
                 (result, createMandateError) -> {
                     if (result != null) {
                         if (URLUtil.isValidUrl(result.getApprovalUrl())) {
-                            braintreeClient.sendAnalyticsEvent(
-                                    "sepa-direct-debit.create-mandate.success");
+                            braintreeClient.sendAnalyticsEvent(SEPADirectDebitAnalytics.CREATE_MANDATE_CHALLENGE_REQUIRED);
                             try {
                                 SEPADirectDebitPaymentAuthRequestParams params =
                                         new SEPADirectDebitPaymentAuthRequestParams(buildBrowserSwitchOptions(result));
+                                braintreeClient.sendAnalyticsEvent(SEPADirectDebitAnalytics.CREATE_MANDATE_SUCCEEDED);
                                 callback.onResult(new SEPADirectDebitPaymentAuthRequest.ReadyToLaunch(params));
                             } catch (JSONException exception) {
-                                braintreeClient.sendAnalyticsEvent(
-                                        "sepa-direct-debit.browser-switch.failure");
-                                callback.onResult(new SEPADirectDebitPaymentAuthRequest.Failure(exception));
+                                braintreeClient.sendAnalyticsEvent(SEPADirectDebitAnalytics.CREATE_MANDATE_FAILED);
```

Comment:
Do we want to add create mandate failed as a param instead of a separate event?

Reply:
Yes I just put these in to make it easier to know what extra info should be added to each of these events - they can be removed when the extra params are added to analytics (separate tickets)

Reply:
But also can remove the entirely if we think it's simpler that way

Reply:
That makes sense, no I think keeping the second event in there will help keep track of the data we need to pass.